new_account.php

```diff
@@ -58,10 +58,12 @@
 		
 		<!-- The form to create a new account -->
 		<div class="account_forms">
-			<!-- php will be used to change link depending on if the user is logged in -->
-			<p class="return_link"><a href="<?php if(!isset($_SESSION['login_user'])) {echo 'login.php';} else {echo 'index.php';} ?>">Return</a></p>
+			<!-- Link that goes back to index page -->
```

Comment:
As commented before this change is because of it previously being a copy of the contact page